src/utilities/gql.ts

```diff
@@ -1503,6 +1755,31 @@ const gql = {
     }
   `,
 
+  GET_PLAN_DERIVATION_GROUP: `#graphql
```

Comment:
Is this query being used? It has a feature permission, but there's no request that's actually making this query.

Reply:
No, we use `SUB_PLAN_DERIVATION_GROUPS` instead of this. Removed and updated the permissions for acknowledgement to reference the subscription instead of this query.